genon/preprocessor/tests/regression/test_hwp_table_structure_regression.py

```
@@ -0,0 +1,624 @@
+
+
+# ---------------------------------------------------------------------------
+# 텍스트 정규화
+# ---------------------------------------------------------------------------
+
+_WS = re.compile(r"\s+")
+
+
+def _norm(text: str) -> str:
+    """공백류(개행 포함)를 단일 스페이스로 축약."""
+    return _WS.sub(" ", (text or "").replace(" ", " ").replace("　", " ")).strip()
```

Ruff (0.16.1)

[warning] String contains ambiguous ` ` (NO-BREAK SPACE). Did you mean ` ` (SPACE)?

(RUF001)